src/haddock/modules/refinement/flexref/defaults.cfg

```diff
@@ -1,200 +1,305 @@
+sinter_cool2_init = 0.001
+sinter_cool2_final = 1.0
+
+# Scaling factors for intermolecular energies for 3rd cooling phase with flexible side-chains
+sinter_cool3_init = 0.05
+sinter_cool3_final = 1.0
+
+# Initial temparature rigid body MD
+temp_high = 2000
+
+# Start/end temperature first SA
+temp_cool1_init = 2000
+temp_cool1_final = 500
+
+# Start/end temperature second SA
+temp_cool2_init = 1000
+temp_cool2_final = 50
+
+# Start/end temperature third SA
+temp_cool3_init = 1000
+temp_cool3_final = 50
+
+
+### Default segIDs
+prot_segid_1 = "A"
+prot_segid_2 = "B"
+prot_segid_3 = "C"
+prot_segid_4 = "D"
+prot_segid_5 = "E"
+prot_segid_6 = "F"
+prot_segid_7 = "G"
+prot_segid_8 = "H"
+prot_segid_9 = "I"
+prot_segid_10 = "J"
+prot_segid_11 = "K"
+prot_segid_12 = "L"
+prot_segid_13 = "M"
+prot_segid_14 = "N"
+prot_segid_15 = "O"
+prot_segid_16 = "P"
+prot_segid_17 = "Q"
+prot_segid_18 = "R"
+prot_segid_19 = "S"
+prot_segid_20 = "T"
+
+
+### Define shape molecules
+shape_1 = false
+shape_2 = false
+shape_3 = false
+shape_4 = false
+shape_5 = false
+shape_6 = false
+shape_7 = false
+shape_8 = false
+shape_9 = false
+shape_10 = false
+shape_11 = false
+shape_12 = false
+shape_13 = false
+shape_14 = false
+shape_15 = false
+shape_16 = false
+shape_17 = false
+shape_18 = false
+shape_19 = false
+shape_20 = false
+
+
+### Distance restraints-related parameters
+
+# force constants
+amb_hot = 10
 amb_cool1 = 10
 amb_cool2 = 50
 amb_cool3 = 50
-amb_firstit = 0
-amb_hot = 10
-c2sym_end1_1 = ""
-c2sym_end1_10 = ""
-c2sym_end1_2 = ""
-c2sym_end1_3 = ""
-c2sym_end1_4 = ""
-c2sym_end1_5 = ""
-c2sym_end1_6 = ""
-c2sym_end1_7 = ""
-c2sym_end1_8 = ""
-c2sym_end1_9 = ""
-c2sym_end2_1 = ""
-c2sym_end2_10 = ""
-c2sym_end2_2 = ""
-c2sym_end2_3 = ""
-c2sym_end2_4 = ""
-c2sym_end2_5 = ""
-c2sym_end2_6 = ""
-c2sym_end2_7 = ""
-c2sym_end2_8 = ""
-c2sym_end2_9 = ""
+unamb_hot = 10
+unamb_cool1 = 10
+unamb_cool2 = 50
+unamb_cool3 = 50
+hbond_hot = 10
+hbond_cool1 = 10
+hbond_cool2 = 50
+hbond_cool3 = 50
+hbonds_on = false
+
+# Random removal of restraints
+noecv = false
+
+# Number of partitions for random removal of restraints
+ncvpart = 2
+
+# Control of the distance restraining potential during the SA
+mrswi_hot = 0.5
+mrswi_cool1 = 0.5
+mrswi_cool2 = 0.5
+mrswi_cool3 = 0.5
+rswi_hot = 0.5
+rswi_cool1 = 0.5
+rswi_cool2 = 0.5
+rswi_cool3 = 0.5
+masy_hot = -1.0
+masy_cool1 = -1.0
+masy_cool2 = -0.1
+masy_cool3 = -0.1
+asy_hot = 1.0
+asy_cool1 = 1.0
+asy_cool2 = 0.1
+asy_cool3 = 0.1
+
+
+### Ab-initio docking parameters
+
+# Center of mass restraints
+cmrest = false
+cmtight = false
+
+# Random surface patches restraints
+ranair = false
+kcont = 1.0
+
+# Surface restraints
+ksurf = 1.0
+surfrest = false
+
+
+### Dihedral angle restraints
+dihedrals_on = false
+dihedrals_hot = 5
+dihedrals_cool1 = 5
+dihedrals_cool2 = 50
+dihedrals_cool3 = 200
+
+
+### Automatic dihedral angle restraining
+# Options are: "none" "alpha" "alphabeta" "all"
+ssdihed = "none"
```

Comment:
Does this need to be NONE for CNS? else would be less prone to confusion if it was `""`

Reply:
none does has a meaning... Wouldn't change the behaviour since we only actually check in the code if that variable is equal to "alpha" "alphabeta" or "all"